src/Microsoft.OData.Cli/GenerateCommand.cs

```diff
@@ -70,45 +91,46 @@ public GenerateCommand()
 
             this.AddOption(ns);
 
-            Option enableTracking = new Option<bool>(new[] { "--enable-tracking", "-et" })
+            Option enableTracking = new Option<bool?>(new[] { "--enable-tracking", "-et" })
             {
                 Name = "enable-tracking",
                 Description = "Enable entity and property tracking."
             };
-            enableTracking.SetDefaultValue(false);
 
+            enableTracking.SetDefaultValue(null);
             this.AddOption(enableTracking);
 
-            Option upperCamelCase = new Option<bool>(new[] { "--upper-camel-case", "-ucc" })
+            Option upperCamelCase = new Option<bool?>(new[] { "--upper-camel-case", "-ucc" })
             {
                 Name = "upper-camel-case",
                 Description = "Disables upper camel casing."
             };
 
+            upperCamelCase.SetDefaultValue(null);
             this.AddOption(upperCamelCase);
 
-            Option internalModifier = new Option<bool>(new[] { "--enable-internal", "-i" })
+            Option internalModifier = new Option<bool?>(new[] { "--enable-internal", "-i" })
             {
                 Name = "enable-internal",
                 Description = "Apply the \"internal\" class modifier on generated classes instead of \"public\" thereby making them invisible outside the assembly."
             };
 
             this.AddOption(internalModifier);
 
-            Option noTimestamp = new Option<bool>(new[] { "--omit-versioning-info", "-vi" })
+            Option noTimestamp = new Option<bool?>(new[] { "--omit-versioning-info", "-vi" })
             {
                 Name = "omit-versioning-info",
                 Description = "Omit runtime version and code generation timestamp from the generated files.",
             };
-            noTimestamp.SetDefaultValue(false);
+            noTimestamp.SetDefaultValue(null);
```

Comment:
```suggestion

            noTimestamp.SetDefaultValue(null);
```
For readability, we usually leave a blank line after a closing brace - unless it's followed by another closing brace.